modules/ROOT/pages/changelog.adoc

```diff
@@ -4,6 +4,15 @@
 
 NOTE: This is the {productname} Community version changelog. For information about the latest {cloudname} or {enterpriseversion} Release, see: xref:release-notes.adoc[{productname} Release Notes].
 
+== 6.8.1 - 2023-11-29
+
+=== Improved
+* Colorpicker now includes the Brightness/Saturation selector and hue slider in the keyboard navigable items.
+
+=== Fixed
+* The functions `schema.isWrapper` and `schema.isInline` didn't exclude element names that started with `#` those should not be considered elements.
```

Comment:
There where some alterations to the master branch on this one: https://github.com/tinymce/tinymce/blob/master/modules/tinymce/CHANGELOG.md

I could merge master into develop but we will do that tomorrow anyway when we do the release.

Reply:
```suggestion
* The functions `schema.isWrapper` and `schema.isInline` did not exclude node names that started with `#` which should not be considered as elements. 
```